js/handlers.js

```diff
@@ -29,6 +29,15 @@ weechat.factory('handlers', ['$rootScope', '$log', 'models', 'plugins', 'notific
         new_date.setHours(0, 0, 0, 0);
         // Check if the date changed
         if (old_date.valueOf() !== new_date.valueOf()) {
+            var supports_formatting_date = false;
+            var should_be_tuesday = new Date(Date.UTC(2012,11,12,3,12,0))
+                .toLocaleDateString("en-US-u-ca-iso8601", {"weekday":"long"});
```

Comment:
At this point, why not just use exceptions instead? This seems all too finicky for my taste